test/load.test.ts

```diff
@@ -35,6 +36,39 @@ const TERM_PKG_RESPONSE = {
       },
       fhirVersion: 'R4',
       url: 'https://packages.simplifier.net/hl7.terminology.r4/1.2.3-test'
+    },
+    '2.0.0': {
+      name: 'hl7.terminology.r4',
+      version: '2.0.0',
+      description: 'None.',
+      dist: {
+        shasum: '1a1467bce19aace45771e0a51ef2ad9c3fe749820',
+        tarball: 'https://packages.simplifier.net/hl7.terminology.r4/2.0.0'
+      },
+      fhirVersion: 'R4',
+      url: 'https://packages.simplifier.net/hl7.terminology.r4/2.0.0'
+    },
+    '2.0.2': {
```

Comment:
This is a little nit-picky, but... this package's `dist-tags` says the `latest` is `1.2.3-test`, so it's a little weird to have versions `2.x` in the manifest. Maybe they can be `1.1.0`, `1.1.1`, and `1.1.2` instead?

Reply:
Whoops. I didn't think about that in context. I can swap them around.

Reply:
[135ecd3](https://github.com/standardhealth/fhir-package-loader/pull/15/commits/135ecd36b1b70c299ba77eafcc19b4726ef91c0d)